src/handlers/componentCreate.ts

```diff
@@ -0,0 +1,104 @@
+import log from '../lib/log';
+import {
+  EXIT_ERROR,
+  EMULSIFY_SYSTEM_CONFIG_FILE,
+  EMULSIFY_PROJECT_CONFIG_FILE,
+} from '../lib/constants';
```

Comment:
This also has to change to `../lib/constants.js` I believe.

Reply:
Basically all of the imports in that format here have to have the .js extension because that is what they get transformed into.

Reply:
@mikeethedude thanks for taking a look at this. I’ve applied the changes. Let me know if any additional changes are required!